client/src/components/PublicPages/AuditPlanner/ElectionResultsCard.tsx

```diff
@@ -189,52 +192,52 @@ const ElectionResultsCard: React.FC<IProps> = ({
                     }
                     intent={errors.candidates?.[i]?.votes && 'danger'}
                   >
-                    <input
-                      aria-label={`Candidate ${i} Votes`}
-                      className={classnames(
-                        Classes.INPUT,
-                        errors.candidates?.[i]?.votes && Classes.INTENT_DANGER
+                    <CandidateVotesInputAndRemoveButtonContainer>
+                      <input
+                        aria-label={`Candidate ${i} Votes`}
+                        className={classnames(
+                          Classes.INPUT,
+                          errors.candidates?.[i]?.votes && Classes.INTENT_DANGER
+                        )}
+                        name={`candidates[${i}].votes`}
+                        onChange={validateAllCandidateVotesFields}
+                        placeholder="0"
+                        readOnly={!editable}
+                        ref={register({
+                          min: {
+                            value: 0,
+                            message: 'Cannot be less than 0',
+                          },
+                          required: 'Required',
+                          validate: () => {
+                            if (
+                              // No need to display this message for all candidate votes inputs
+                              i === 0 &&
+                              getValues().candidates.every(
+                                candidate => (candidate.votes || 0) <= 0
+                              )
+                            ) {
+                              return 'At least 1 candidate must have greater than 0 votes'
+                            }
+                            return true
+                          },
+                          valueAsNumber: true,
+                        })}
+                        type="number"
+                      />
+                      {editable && (
```

Comment:
I like this new arrangement!

I feel like we're still losing the structure in readonly mode because we lose the shape of the input fields. What do you think of something like this?

<img width="685" alt="Screen Shot 2022-09-15 at 6 13 13 PM" src="https://user-images.githubusercontent.com/530106/190536094-008315a7-18b1-40c8-82b4-91480db6353b.png">

This is a fill of `LIGHT_GRAY5`. We could also experiment with a stroke. I also played with removing the intent from the remove buttons to differentiate their state.

Reply:
Nice, I like the screenshot! Incorporated ✅